apps/gportal/vite.config.js

```diff
@@ -21,7 +23,32 @@ export default defineConfig(() => {
     },
     base: "",
     server: {
-      proxy: require("../dev-proxy.config"),
+      proxy: {
```

Comment:
Why did you make your own proxy?
I mean, if this of a general use then please push to the shared proxy

Reply:
I will create a new PR for this.